test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -644,6 +644,16 @@ def func(tensor):
         tensor = torch.view_as_complex(torch.randn(2, 1025, 400, 2))
         self._assert_consistency_complex(func, tensor)
 
+    def test_compute_mvdr_weights_souden(self):
+        def func(_):
+            channel = 4
+            psd_speech = torch.rand(129, channel, channel, dtype=torch.cfloat)
+            psd_noise = torch.rand(129, channel, channel, dtype=torch.cfloat)
+            return F.compute_mvdr_weights_souden(psd_speech, psd_noise, 0)
+
+        dummy = torch.rand(1, 1)
+        self._assert_consistency(func, dummy)
```

Comment:
Please update the test so that the function receives some input before/after scripted.